pandas/core/groupby/generic.py

```diff
@@ -470,10 +470,9 @@ def _aggregate_named(self, func, *args, **kwargs):
 
     __examples_series_doc = dedent(
         """
-    >>> ser = pd.Series(
-    ...    [390.0, 350.0, 30.0, 20.0],
-    ...    index=["Falcon", "Falcon", "Parrot", "Parrot"],
-    ...    name="Max Speed")
+    >>> ser = pd.Series([390.0, 350.0, 30.0, 20.0],
+    ...                 index=["Falcon", "Falcon", "Parrot", "Parrot"],
+    ...                 name="Max Speed")
```

Comment:
Just curious - what is the format that's being enforced here? Would be great if it was `black`'s, but happy with this too.

Reply:
No format is being enforced here currently. We're just calling flake8 on a tempfile of this content. 

It would be really nice in the future if we could autoformat this as well.